README.md

```diff
@@ -2,3 +2,39 @@ Instant Share
 =============
```

Comment:
One thing to consider from a user's perspective is that the main problem this attempts to address (not knowing how to set up InstantShare) is still not addressed. Is there anything that should be added to explain how to point the client at the specific server instance and how to build the client?